daffodil-core/src/main/scala/org/apache/daffodil/core/runtime1/SchemaSetRuntime1Mixin.scala

```diff
@@ -68,7 +68,7 @@ trait SchemaSetRuntime1Mixin {
   }.value
 
   def onPath(xpath: String): DFDL.DataProcessor = {
-    Assert.usage(!isError)
+    Assert.usage(!isError, getDiagnostics.mkString)
```

Comment:
Same comment as above, but about the `onPath` function instead of `forLanguage`.